x-pack/plugins/security_solution/server/lib/detection_engine/routes/signals/set_alert_assignees_route.ts

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { transformError } from '@kbn/securitysolution-es-utils';
+import { uniq } from 'lodash/fp';
+import type { SetAlertAssigneesRequestBodyDecoded } from '../../../../../common/api/detection_engine/alert_assignees';
+import { setAlertAssigneesRequestBody } from '../../../../../common/api/detection_engine/alert_assignees';
+import type { SecuritySolutionPluginRouter } from '../../../../types';
+import {
+  DEFAULT_ALERTS_INDEX,
+  DETECTION_ENGINE_ALERT_ASSIGNEES_URL,
+} from '../../../../../common/constants';
+import { buildSiemResponse } from '../utils';
+import { buildRouteValidation } from '../../../../utils/build_validation/route_validation';
+import { validateAlertAssigneesArrays } from './helpers';
+
+export const setAlertAssigneesRoute = (router: SecuritySolutionPluginRouter) => {
```

Comment:
We need to manually add the OpenAPI specs for the new routes.

Reply:
Added PR for this https://github.com/elastic/kibana/pull/172285